include/deal.II/grid/grid_tools.h

```diff
@@ -1361,25 +1362,33 @@ namespace GridTools
                                              DynamicSparsityPattern             &connectivity);
 
   /**
-   * Use the METIS partitioner to generate a partitioning of the active cells
+   * Use METIS or ZOLTAN partitioner to generate a partitioning of the active cells
```

Comment:
to make it extendable, it's probably easier to say

> Use graph partitioner...

and below

> Currently the user can choose between METIS (default) and ZOLTAN partitioner. If deal.II was not
> compiled with the corresponding package, then the function will generate en error unless .... .

this is in line of what @bangerth suggested below https://github.com/dealii/dealii/pull/5514/files#r152897649